<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.</comment>
 <class>DCC_ConfigUI</class>
 <widget class="QWidget" name="DCC_ConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCC_Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="dccPathLbl">
         <property name="text">
          <string>Download &amp;folder:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="buddy">
          <cstring>kcfg_DccPath</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KUrlRequester" name="kcfg_DccPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccAddPartner">
       <property name="text">
        <string>&amp;Add sender to file name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccCreateFolder">
       <property name="text">
        <string>Cr&amp;eate folder for sender</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccSpaceToUnderscore">
       <property name="text">
        <string>Convert spaces in file names to underscores before sending</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccAutoGet">
       <property name="text">
        <string>Automaticall&amp;y accept download offer</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccAutoResume">
       <property name="text">
        <string>Au&amp;tomatically resume download</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccFastSend">
       <property name="text">
        <string>Enable fast DCC send (Might &amp;not work with all clients)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccPassiveSend">
       <property name="text">
        <string>Enable passive DCC send</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccChatAutoAccept">
       <property name="text">
        <string>Automatically accept DCC chat offer</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QCheckBox" name="kcfg_DccUPnP">
       <property name="text">
        <string>Enable UPnP for NAT traversal</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="kcfg_DccIPv4Fallback">
         <property name="whatsThis">
          <string>If this option enabled dcc transfers for IPv6 connections will be done over IPv4 interface set here</string>
         </property>
         <property name="text">
          <string>Fallback to an IPv&amp;4 interface for DCC send:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="kcfg_DccIPv4FallbackIface">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="textLabel7">
       <property name="text">
        <string>DCC send t&amp;imeout:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>kcfg_DccSendTimeout</cstring>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="KPluralHandlingSpinBox" name="kcfg_DccSendTimeout">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="textLabel6">
       <property name="text">
        <string>Buffer si&amp;ze:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>kcfg_DccBufferSize</cstring>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="KPluralHandlingSpinBox" name="kcfg_DccBufferSize">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="singleStep">
        <number>128</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>IP</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>&amp;Method to get own IP:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_DccMethodToGetOwnIp</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_DccMethodToGetOwnIp"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ownIP">
        <property name="text">
         <string>O&amp;wn IP:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_DccSpecificOwnIp</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="kcfg_DccSpecificOwnIp"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Ports</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="kcfg_DccSpecificSendPorts">
        <property name="text">
         <string>Enable specific &amp;ports for DCC transfer server:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="kcfg_DccSendPortsFirst">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1026</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>to</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_DccSendPortsLast">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1027</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="kcfg_DccSpecificChatPorts">
        <property name="text">
         <string>Enable specific p&amp;orts for DCC chat:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSpinBox" name="kcfg_DccChatPortsFirst">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1026</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>to</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_DccChatPortsLast">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1027</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer4_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kpluralhandlingspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_DccPath</tabstop>
  <tabstop>kcfg_DccAddPartner</tabstop>
  <tabstop>kcfg_DccCreateFolder</tabstop>
  <tabstop>kcfg_DccSpaceToUnderscore</tabstop>
  <tabstop>kcfg_DccAutoGet</tabstop>
  <tabstop>kcfg_DccAutoResume</tabstop>
  <tabstop>kcfg_DccFastSend</tabstop>
  <tabstop>kcfg_DccPassiveSend</tabstop>
  <tabstop>kcfg_DccChatAutoAccept</tabstop>
  <tabstop>kcfg_DccUPnP</tabstop>
  <tabstop>kcfg_DccIPv4Fallback</tabstop>
  <tabstop>kcfg_DccIPv4FallbackIface</tabstop>
  <tabstop>kcfg_DccSendTimeout</tabstop>
  <tabstop>kcfg_DccBufferSize</tabstop>
  <tabstop>kcfg_DccMethodToGetOwnIp</tabstop>
  <tabstop>kcfg_DccSpecificOwnIp</tabstop>
  <tabstop>kcfg_DccSpecificSendPorts</tabstop>
  <tabstop>kcfg_DccSendPortsFirst</tabstop>
  <tabstop>kcfg_DccSendPortsLast</tabstop>
  <tabstop>kcfg_DccSpecificChatPorts</tabstop>
  <tabstop>kcfg_DccChatPortsFirst</tabstop>
  <tabstop>kcfg_DccChatPortsLast</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_DccSpecificSendPorts</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_DccSendPortsFirst</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_DccSpecificSendPorts</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel1_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_DccSpecificSendPorts</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_DccSendPortsLast</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_DccSpecificChatPorts</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_DccChatPortsFirst</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_DccSpecificChatPorts</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel2_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_DccSpecificChatPorts</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_DccChatPortsLast</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_DccIPv4Fallback</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_DccIPv4FallbackIface</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
